scipy/stats/stats.py

```diff
@@ -4218,7 +4218,8 @@ def ks_2samp(data1, data2):
     return Ks_2sampResult(d, prob)
 
 
-def mannwhitneyu(x, y, use_continuity=True):
+def mannwhitneyu(x, y, use_continuity=True, use_exact=None,
```

Comment:
It looks like `None` acts like 'auto' -- the exactness will be determined by data size.